packages/pigeon/platform_tests/android_kotlin_unit_tests/android/app/src/test/java/com/example/android_kotlin_unit_tests/PrimitiveTest.kt

```diff
@@ -0,0 +1,434 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Instead of rewriting all these tests in kotlin, couldn't we just run the java tests against the kotlin generated code?

Reply:
I've been trying this, but unfortunately I'm not having success.
Mainly, because Kotlin deal with static method in a different way than Java. To call a static method from Kotlin in Java it's needed use the `Companion` keyword. Changing this, the Java tests would be broken.

Despite this, another minor things like the name of classes that Java generate:
- list.dart -> PigeonList
- android_unittests.dart -> Pigeon
- ...

What do you recommend to me in order to proceed this question?